index.d.ts

```diff
@@ -175,6 +182,9 @@ declare module "react-native-twilio-video-webrtc" {
       videoBitrate?: number;
     };
     enableRemoteAudio?: boolean;
+    encodingParameters?: {
+      enableH264Codec?: boolean;
+    };
```

Comment:
It looks like we're defining `encodingParameters` twice now?

Reply:
ah right 🤦 
removing it.